dashboard/app/models/user_permission.rb

```diff
@@ -32,10 +32,9 @@ class UserPermission < ActiveRecord::Base
     # Also grants access to viewing extra links related to editing these.
     # Also makes the account satisfy authorized_teacher?.
     LEVELBUILDER = 'levelbuilder'.freeze,
-    # Grants access to reseting (to 0) the abuse score for projects,
-    # and blocking and unblocking legacy shares.
-    RESET_ABUSE = 'reset_abuse'.freeze,
     # Grants ability to (un)feature projects in the the public project gallery.
+    # Also, grants access to resetting (to 0) the abuse score for projects,
+    # and blocking and unblocking legacy shares (formerly RESET_ABUSE).
```

Comment:
Nice comment, this will hopefully help people track down this change if needed in the future.